public/partials/io/import.html

```diff
@@ -99,6 +124,14 @@ <h5>Created items</h5>
             No new item has been created.
         </div>
 
+        <div ng-if="created.length">
+            <ul>
+                <li ng-repeat="item in updated">
```

Comment:
- The length of `created` is checked and then we iterate over `updated`. Problem ?
- `updated` does not exist in the scope.